extensions/text-shortcuts/src/create-shortcut.tsx

```diff
@@ -0,0 +1,406 @@
+            <Form.Dropdown
+              id={"case" + index}
+              key={"case" + index}
+              title={TactionType.CASE}
+              defaultValue={array[index].content[0]}
+              onChange={async (newValue) => {
+                updateTactionContent(newValue, index, [...array], setTactions);
+              }}
+            >
+              {cases.map((cases, caseIndex) => {
+                return <Form.Dropdown.Item key={"case" + caseIndex} title={cases} value={cases} />;
+              })}
+            </Form.Dropdown>
+          </React.Fragment>
+        );
+      }
+      case TactionType.CODER: {
+        return (
+          <React.Fragment key={"coder_fragment" + index}>
+            <Form.Separator />
+            <Form.Dropdown
+              id={"coder" + index}
+              key={"coder" + index}
+              title={TactionType.CODER}
+              defaultValue={array[index].content[0]}
+              onChange={async (newValue) => {
+                updateTactionContent(newValue, index, [...array], setTactions);
+              }}
+            >
+              {coders.map((coder) => {
+                return <Form.Dropdown.Item key={"coder" + coder} title={coder} value={coder} />;
+              })}
+            </Form.Dropdown>
+          </React.Fragment>
+        );
+      }
+      case TactionType.TRANSFORM: {
+        return (
+          <React.Fragment key={"transform_fragment" + index}>
+            <Form.Separator />
+            <Form.Dropdown
+              id={"transform" + index}
+              key={"transform" + index}
+              title={TactionType.TRANSFORM}
+              defaultValue={array[index].content[0]}
+              onChange={async (newValue) => {
+                updateTactionContent(newValue, index, [...array], setTactions);
+              }}
+            >
+              {transforms.map((transform) => {
+                return <Form.Dropdown.Item key={"convert" + transform} title={transform} value={transform} />;
+              })}
+            </Form.Dropdown>
+          </React.Fragment>
+        );
+      }
+    }
+  });
+}
+
+function CreateShortcutActions(props: {
+  info: ShortcutInfo;
+  tactions: Taction[];
+  localShortcuts: Shortcut[];
+  setTactions: React.Dispatch<React.SetStateAction<Taction[]>>;
+  updateListUseState: [number[], React.Dispatch<React.SetStateAction<number[]>>];
+}) {
+  const info = props.info;
+  const tactions = props.tactions;
+  const localShortcuts = props.localShortcuts;
+  const setTactions = props.setTactions;
+  const [updateList, setUpdateList] = props.updateListUseState;
+  const { pop } = useNavigation();
+  return (
+    <ActionPanel>
+      <Action
+        title="More Actions"
+        icon={Icon.Terminal}
+        onAction={async () => {
+          try {
+            await simulatePressKeyboard(40, "command");
+          } catch (e) {
+            await showToast(Toast.Style.Failure, String(e));
+          }
+        }}
+      />
```

Comment:
```suggestion
```

Raycast already has a global for opening the action panel (`⌘K`), so creating an action for it is superfluous. Additionally, making it he primary action can mess with user expectation: I'd expect the primary action on a `Form` to…submit the form. 😸

Reply:
Okay, here I was mainly worried that the user would create a shortcut directly without adding an action, so I wrote it this way.
But now I've added help text and a hint to detect if the action is included, so I'll change it here